Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+pipeline {
+    agent {
+        docker {
+            image 'microsoft/azure-cli'
+            args '-u root:root -v /var/run/docker.sock:/var/run/docker.sock'
+        }
+    }
+    stages {
+        stage('Setup building environment') {
+            steps {
+                sh 'apk upgrade && apk update && apk add --no-cache go make musl-dev docker'
```

Comment:
Can you please add a comment here indicating what the building of musl-dev means and also where this would get executed ?